pycbc/waveform/waveform.py

```diff
@@ -1167,7 +1167,8 @@ def td_fd_waveform_transform(approximant):
         # We can also make a td version of this
         # This will override any existing approximants with the same name
         # (ex. IMRPhenomXX)
-        cpu_td[approximant] = get_td_waveform_from_fd
+        if approximant not in td_apx:
```

Comment:
@yi-fan-wang I don't think we can approve this. In particular, the lalsim internal conversions don't preserve phase / timing in as exact a way. This has caused several problems in the past with inconsistencies between templates and injections. If you want to be able to select between the path, I think having a way to do that would be fine, but please do not change the default here.

Reply:
@yi-fan-wang Yeah, I think addresses if you want to allow either path is probably outside the scope of this PR, so if needed consider what to do separately.